extensions/oplogPopulator/modules/Connector.js

```diff
@@ -170,19 +184,54 @@ class Connector {
         return JSON.stringify(pipeline);
     }
 
+    /**
+     * Handles updating the values of _bucketsGotModified
+     * @param {boolean} bucketsGotModified value of _state.bucketsGotModified
+     * to set
+     * @param {number} timeBeforeUpdate time just before async
+     * connector update operation (only needed when updated connector)
+     * @returns {undefined}
+     */
+    _updateConnectorState(bucketsGotModified, timeBeforeUpdate = null) {
```

Comment:
Nit:
```suggestion
    _updateConnectorState(bucketsGotModified, timeBeforeUpdate = 0) {
```

to be consistent with the jsdoc type